specification/communication/data-plane/Sms/preview/2024-01-14-preview/communicationservicessms.json

```
@@ -0,0 +1,416 @@
+        }
+      }
+    },
+    "MmsSendResponse": {
+      "description": "Response for a successful or multi status MMS send request.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MmsSendResponseItem"
+          }
+        }
+      }
+    },
+    "MmsSendResponseItem": {
+      "description": "MMS Response for a single recipient.",
+      "required": [
+        "httpStatusCode",
+        "successful",
+        "to"
+      ],
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The recipient's phone number in E.164 format.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "messageId": {
+          "description": "The identifier of the outgoing message. Only present if message processed.",
+          "type": "string"
+        },
+        "httpStatusCode": {
+          "format": "int32",
+          "description": "HTTP Status code.",
+          "type": "integer"
+        },
+        "errorMessage": {
+          "description": "Optional error message in case of 4xx/5xx/repeatable errors.",
+          "type": "string"
+        },
+        "repeatabilityResult": {
+          "description": "The result of a repeatable request with one of the case-insensitive values accepted or rejected.",
+          "type": "string"
+        },
+        "successful": {
+          "description": "Indicates if the message is processed successfully or not.",
+          "type": "boolean"
+        }
+      }
+    },
+    "SendMessageRequest": {
+      "description": "Represents the properties of a send SMS message request.",
+      "required": [
+        "from",
+        "message",
+        "smsRecipients"
+      ],
+      "type": "object",
+      "properties": {
+        "from": {
+          "description": "The sender's identifier(typically phone number in E.164 format) that is owned by the authenticated account.",
```

Why only "typically"? I thought it always needs to be in E.164

```suggestion
          "description": "The sender's identifier (typically phone number in E.164 format) that is owned by the authenticated account.",
```
(and missing space)

---

And can it be a different identifier than a phone number?

---

Sender can be also alphanumeric. https://learn.microsoft.com/en-us/azure/communication-services/concepts/sms/concepts#sender-types-supported

---

```suggestion
          "description": "The sender's identifier (phone number in E.164 format, short code or alphanumeric sender ID) that is owned by the authenticated account.",
```